api/schema/v1/modules/packetio.yaml

```diff
@@ -361,6 +367,49 @@ definitions:
                 description: Client identifier
         required:
           - method
+      ipv6:
+        type: object
+        description: IPv6 protocol configuration
+        properties:
+          method:
+            type: string
+            description: Configuration method
+            enum:
+              - static
+              - dhcp6
+              - auto
+          link_local_address:
+            type: string
+            description: IPv6 link local address
+          static:
+            type: object
+            description: Static configuration parameters
+            properties:
+              address:
+                type: string
+                description: IPv6 address
+              prefix_length:
+                type: integer
+                description: Prefix length
+                format: int32
+              gateway:
+                type: string
+                description: Default gateway
+            required:
+              - address
+              - prefix_length
+          dhcp6:
+            type: object
+            description: DHCPv6 configuration parameters
+            properties:
+              stateless:
```

Comment:
Setting "stateless" to false currently cause an error because lwip built-in DHCPv6 doesn't support stateful mode.